site/content/docs/monitoring/global-locations.md

```diff
@@ -25,3 +25,22 @@ Current locations in Checkly are
 {{< info >}}
 Learn more about <a href="/docs/private-locations/private-locations-getting-started/">private locations</a> to monitor your private and segregated applications and APIs.
 {{< /info >}}
+
+## Scheduling strategies
+
+Checkly provides two scheduling strategies for running API, Browser or Multistep checks: Round-robin or Parallel scheduling. 
+To select a scheduling strategy go to ‘Scheduling and locations’ when creating or editing a check.
+
+### Round-robin
+
+Using round-robin, your check will run on one of the selected locations each time it is scheduled. The next check run will be scheduled on a different location from the list until all locations have been run once, and the check rotates back to the first location in the list.
+
+When using the round-robin strategy you can choose to have retries to run from a random location of the ones selected, or run it from the same location as the first attempt
+
+Use round-robin when the service you are monitoring can be considered available as long as at least one location is available and detecting a regional outage is not critical.
+
+### Parallel
```

Comment:
Below this block, let's add a disclaimer about switching between both strategies as we did here https://github.com/checkly/checkly-webapp/pull/6205

Reply:
Also, might be cool to add a `beta` badge right to this text, however, not sure if is possible haha

Reply:
Let's find out!

Reply:
@maxigimenez no one will find out about this in the docs. We need to have a small notice in the switched in the app if we want to inform people about the "metric switch"

Reply:
@tnolet - It's in the app already, Maxi suggested we add it to the docs as well.

![Screenshot 2023-12-14 at 09 33 33](https://github.com/checkly/docs.checklyhq.com/assets/11697647/6a37772a-fad0-41b8-95e0-65cb8fe776d8)

Reply:
@tnolet We have that one already in the webapp, adding it is nice in case someone asks on support, we can link it directly to the docs